### PR TITLE
Update wording on GitHub Organisation Invitation.

### PR DESCRIPTION
**Pull request by vijaykannan21**
Update wording on GitHub Organisation Invitation

- The page title ("GitHub Organisation Invitation") doesn't explain what this page is for.  change to "Verify your work email address.
- Replace "To access the Ministry of Justice GitHub organisations you've selected, your email address @justice.gov.uk will be verified using Azure Active Directory (AD). 
- This secure step is essential to confirm your identity and initiate the process of joining our GitHub organisations
- By clicking the button below, you will be redirected to a secure sign-on page where you will be asked to confirm your Office 365 login details
- "After successful verification a GitHub invitation will be sent to your work email address. You will need to accept this invitation to join the selected GitHub organisation
- Replace button text "Authenticate" to verify

### CODE REVIEW

**Review by AntonyBishop — CHANGES_REQUESTED** [review]
This needs to be removed as per the comments in the related issue

**Review by AntonyBishop — CHANGES_REQUESTED** [review]
Remove trailing "

**Review by AntonyBishop — CHANGES_REQUESTED** [review]
Needs to be removed as per the related issue comments

**Review by AntonyBishop — CHANGES_REQUESTED** [review]
The button just needs to say verify. The email address on the button is not required.

File: app/templates/pages/justice-and-other-user.html [hunk 1]
```diff
@@ -13,26 +13,28 @@
 {% endblock %}
 
 {% block content %}
-<h1 class="govuk-heading-l">GitHub Organisation Invitation</h1>
+<h1 class="govuk-heading-l">Verify your work email address</h1>
 
 <p class="govuk-body">
   To access the Ministry of Justice GitHub organisations you've selected, your email address <strong>{{email}}</strong>
```
Comment (AntonyBishop) [inline app/templates/pages/justice-and-other-user.html:19]: This section needs to be replaced by "This secure step is essential to confirm your identity and initiate the process of joining our GitHub organisations."

File: app/templates/pages/justice-and-other-user.html [hunk 2]
```diff
@@ -13,26 +13,28 @@
...
 
 <p class="govuk-body">
-  By clicking the button below, you will be redirected to a secure Azure AD login page for authentication.
+  By clicking the button below, you will be redirected to a secure sign-on page where you will be asked to confirm your Office 365 login details.
 </p>
 
 <p class="govuk-body">
-  After successfully authenticating with your @justice.gov.uk credentials, GitHub will send an invitation to
-  <strong>{{email}}</strong>. You will need to accept this invitation to join the selected GitHub organisation.
+  After successfully authenticating with your @justice.gov.uk credentials, GitHub will send an invitation to
```
Comment (AntonyBishop) [inline app/templates/pages/justice-and-other-user.html:29]: This section needs to be replaced with "After successful verification a GitHub invitation will be sent to your work email address. You will need to accept this invitation to join the selected GitHub organisation."

**Review by AntonyBishop — CHANGES_REQUESTED** [review]

**Review by vijaykannan21 — COMMENTED** [review]
Updated the changes

File: app/templates/pages/justice-and-other-user.html [hunk 3]
```diff
@@ -13,26 +13,28 @@
...
 {% block content %}
-<h1 class="govuk-heading-l">GitHub Organisation Invitation</h1>
+<h1 class="govuk-heading-l">Verify your work email address</h1>
 
 <p class="govuk-body">
-  To access the Ministry of Justice GitHub organisations you've selected, your email address <strong>{{email}}</strong>
-  will be verified
-  using Azure Active Directory (AD). This secure step is essential to confirm your identity and initiate the process of
-  joining our GitHub organisations.
+  This secure step is essential to confirm your identity and initiate the process of joining our GitHub organisations. <strong>{{email}}</strong>
```
Comment (AntonyBishop) [inline app/templates/pages/justice-and-other-user.html:19]: This paragraph still contains the old text. That needs to be removed. You need to remove everything from 

> <strong>{{email}}</strong> will be verified using Azure Active Directory (AD).
>   This secure step is essential to confirm your identity and initiate the process of joining our GitHub organisations.

The paragraph should just read:

> This secure step is essential to confirm your identity and initiate the process of joining our GitHub organisations.

File: app/templates/pages/justice-and-other-user.html (hunk 2 above)
Comment (AntonyBishop) [reply app/templates/pages/justice-and-other-user.html:29]: Line 29 still contains the old text:

> After successfully authenticating with your @justice.gov.uk credentials, GitHub will send an invitation to <strong>{{email}}</strong>. You will need to accept this invitation to join the selected GitHub organisation.

It should just read:

> After successful verification a GitHub invitation will be sent to your work email address. You will need to accept this invitation to join the selected GitHub organisation.

**Review by AntonyBishop — CHANGES_REQUESTED** [review]
See comments inline with changes

File: app/templates/pages/justice-and-other-user.html (hunk 3 above)
Comment (vijaykannan21) [reply app/templates/pages/justice-and-other-user.html:19]: yup i just removed the duplicate line.

Comment (AntonyBishop) [reply app/templates/pages/justice-and-other-user.html:19]: After the first line it still says "<strong>{{email}}</strong>". Please remove.

**Comment by vijaykannan21** [comment]
Unit test got failed so closing the PR

**Pull request closed without merging**